modules/rostests/apitests/ntdll/NtReadFile.c

```diff
@@ -64,8 +64,6 @@ START_TEST(NtReadFile)
     trace("System is %d bits, Size of MDL: %lu\n", Is64BitSystem() ? 64 : 32, SizeOfMdl());
     trace("Max MDL data size: 0x%lx bytes\n", LargeMdlMaxDataSize);
 
-    ByteOffset.QuadPart = 0;
```

Comment:
This function follows a "ByteOffset is 0 by default" approach. Notice how changes to ByteOffset are done directly before individual NtReadFile/NtWriteFile calls, and reset to 0 happens immediately after. So if you want to change this, every read/write should have the byte offset set explicitly, directly before the call.

Reply:
Reverted.